app/src/main/res/layout-land/activity_edit.xml

```diff
@@ -0,0 +1,129 @@
+        android:layout_height="wrap_content"
+        android:textAppearance="?android:attr/textAppearanceMedium"
+        android:text="@string/StartTimeLabel"
+        android:id="@+id/StartTimeLabel"
+        android:layout_alignParentTop="true"
+        android:layout_alignParentLeft="true"
+        android:layout_centerHorizontal="true"
+        android:layout_toLeftOf="@+id/StartDatePicker"
+        android:layout_alignTop="@+id/StartTimePicker"
+        android:layout_alignBottom="@+id/StartTimePicker"/>
+
+    <DatePicker
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/StartDatePicker"
+        android:datePickerMode="spinner"
+        android:layout_alignParentTop="true"
+        android:clickable="false"
+        android:calendarViewShown="false"
+        android:spinnersShown="true"
+        android:focusable="false"
+        android:layout_alignTop="@+id/StartTimePicker"
+        android:layout_alignBottom="@+id/StartTimePicker"
+        android:layout_toLeftOf="@+id/StartTimePicker"/>
+
+    <TimePicker
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/StartTimePicker"
+        android:timePickerMode="spinner"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentTop="true"
+        android:scrollIndicators="bottom|top"
+        android:clickable="true"
+        android:longClickable="true"
+        android:measureAllChildren="false"
+        />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textAppearance="?android:attr/textAppearanceMedium"
+        android:text="@string/EndTimeLabel"
+        android:id="@+id/EndTimeLabel"
+        android:layout_below="@+id/StartTimeLabel"
+        android:layout_alignParentLeft="false"
+        android:layout_alignTop="@+id/EndTimePicker"
+        android:layout_alignBottom="@+id/EndTimePicker"
+        android:layout_toLeftOf="@+id/EndDatePicker"
+        android:layout_alignLeft="@+id/StartTimeLabel"/>
+
+    <DatePicker
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/EndDatePicker"
+        android:spinnersShown="true"
+        android:datePickerMode="spinner"
+        android:layout_centerHorizontal="true"
+        android:layout_below="@+id/StartDatePicker"
+        android:calendarViewShown="false"
+        android:layout_alignTop="@+id/EndTimePicker"
+        android:layout_alignBottom="@+id/EndTimePicker"
+        android:layout_toLeftOf="@+id/EndTimePicker"/>
+
+    <TimePicker
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/EndTimePicker"
+        android:timePickerMode="spinner"
+        android:layout_alignParentRight="true"
+        android:layout_below="@+id/StartTimePicker"
+        />
+
+    <TextView
```

Comment:
Na das Feedback ist aber etwas unfair. Wenn ich die Möglichkeit hätte im Emulator mir anzuschauen wie der Dialog tatsächlich gerendered wird, würde ich das einsehen. Da wir das aber - wie Sie mir in der Mail hierzu gesagt haben - auch noch gar nicht können sollen, kann ich hier dann aber auch leider nur mit den Schultern zucken.